### PR TITLE
Signoz cloud onboarding v1

### PR DESCRIPTION
**Pull request by YounixM**

### CODE REVIEW

**Comment by YounixM** [comment]
Approved here: 

**https://github.com/YounixM/signoz/pull/1**


Minor review comments on the PR, to be addressed in subsequent PR's.

File: frontend/src/AppRoutes/index.tsx [hunk 1]
```diff
@@ -1,20 +1,72 @@
...
+				(route) => route?.key !== ROUTES.GET_STARTED,
+			);
+
+			setRoutes(newRoutes);
+		}
+
+		if (isLoggedInState && isChatSupportEnabled) {
+			// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+			// @ts-ignore
+			window.Intercom('boot', {
```
Comment (palashgdev) [inline frontend/src/AppRoutes/index.tsx:63]: need to fix this one

File: frontend/src/constants/localStorage.ts [hunk 2]
```diff
@@ -11,4 +11,7 @@ export enum LOCALSTORAGE {
 	GRAPH_VISIBILITY_STATES = 'GRAPH_VISIBILITY_STATES',
 	TRACES_LIST_COLUMNS = 'TRACES_LIST_COLUMNS',
 	LOGS_LIST_COLUMNS = 'LOGS_LIST_COLUMNS',
+	LOGGED_IN_USER_NAME = 'LOGGED_IN_USER_NAME',
```
Comment (YounixM) [inline frontend/src/constants/localStorage.ts:14]: Need to remove these constants as they are not being consumed now

**Review by palashgdev — APPROVED** [review]
LGTM but will address small comments in next PR

**Pull request merged**